velox/common/memory/SharedArbitrator.h

```diff
@@ -90,29 +92,70 @@ class SharedArbitrator : public memory::MemoryArbitrator {
   // The kind string of shared arbitrator.
   inline static const std::string kind_{"SHARED"};
 
+  struct ArbitrationOperation {
+    MemoryPool* const requestPool;
+    MemoryPool* const requestRoot;
+    const std::vector<std::shared_ptr<MemoryPool>>& candidatePools;
+    const uint64_t targetBytes;
+    const std::chrono::steady_clock::time_point startTime;
+
+    std::vector<Candidate> candidates;
+
+    bool needGlobalArbitration{false};
+    uint32_t numGlobalArbitrationAttempts{0};
+    uint64_t waitTimeUs{0};
+
+    explicit ArbitrationOperation(
+        uint64_t targetBytes,
+        const std::vector<std::shared_ptr<MemoryPool>>& candidatePools)
```

Comment:
Should we also prefix _ to the arg names to align with the style (although not needed syntax wise)

Reply:
This might not required as it calls the other ctor to init? Thanks!